Sources/RecurrenceRule.swift

```diff
@@ -29,7 +29,7 @@ public struct RecurrenceRule {
     /// The start date of recurrence rule.
     ///
     /// The default value of this property is current date.
-    public var startDate = Date()
+    public var startDate: Date?
```

Comment:
when generating rrule for recurring to do, we don't need DTSTART since we rely on schedule date instead for correct Date generation.
This change allows not setting DTSTART